common/testing/mocksdk/generate.sh

```diff
@@ -5,18 +5,16 @@
 
 # These mocks need to be manually fixed up after generation because gomock
 # uses the types in the internal package instead of the public type aliases.
-mockgen -copyright_file ../../../LICENSE -package "$GOPACKAGE" go.temporal.io/sdk/client Client | sed \
-	-e 's,\<internal\>,client,g' \
-	-e '/"go.temporal.io\/sdk\/converter"/d' \
-	-e 's,client "go.temporal.io/sdk/client",client "go.temporal.io/sdk/client"\
-	converter "go.temporal.io/sdk/converter",' \
-	> client_mock.go
-mockgen -copyright_file ../../../LICENSE -package "$GOPACKAGE" go.temporal.io/sdk/worker Worker | sed \
-	-e 's,internal.RegisterWorkflowOptions,workflow.RegisterOptions,g' \
-	-e 's,internal.RegisterActivityOptions,activity.RegisterOptions,g' \
-	-e 's,internal "go.temporal.io/sdk/internal",activity "go.temporal.io/sdk/activity"\
-	workflow "go.temporal.io/sdk/workflow",' \
-	> worker_mock.go
-mockgen -copyright_file ../../../LICENSE -package "$GOPACKAGE" go.temporal.io/sdk/client WorkflowRun | sed \
-	-e 's,\<internal\>,client,g' \
-	> workflowrun_mock.go
+mockgen -copyright_file ../../../LICENSE -package "$GOPACKAGE" go.temporal.io/sdk/client Client | \
+  sed -e 's,internal,client,g' | \
+  goimports > client_mock.go
+
+mockgen -copyright_file ../../../LICENSE -package "$GOPACKAGE" go.temporal.io/sdk/worker Worker | \
+  sed -e 's,internal.RegisterWorkflowOptions,workflow.RegisterOptions,g' \
+      -e 's,internal.RegisterActivityOptions,activity.RegisterOptions,g' \
+      -e 's,internal "go.temporal.io/sdk/internal",activity "go.temporal.io/sdk/activity"\n\tworkflow "go.temporal.io/sdk/workflow",' | \
```

Comment:
I was under the impression that `\n` didn't work on macos sed. If you're not using backslash+newline anymore, you can remove the shellcheck disable at the top of the file